### ISSUE TITLE
HTTPS breaks the fb iframe 

### ISSUE THREAD

**Issue by tonyatoms** [tonyatoms/MPDC-site#2]
well, it is the insecurity of the iframe that is actually the problem...

1. Try adjusting the URL?

**Comment by tonyatoms** [comment]
seems to work now...

**Comment by tonyatoms** [comment]
works in Safari (11.0), and Chrome (62.0.3202.94 (Official Build) (64-bit) )
but not Firefox (57.0). so, uh, probably won't fix

**Comment by tonyatoms** [comment]
Not broken in FF, but blocked by default(?) config:
`The resource at “https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2FMcKParkDC%2F&tabs=timeline&width=440&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=false&appId=758860070988571” was blocked because tracking protection is enabled`
